framework/wazuh/tests/test_agent.py

```diff
@@ -1593,3 +1593,11 @@ def test_unset_single_group_agent_ko(agent_basic_mock, group_exists_mock, get_gr
     with pytest.raises(WazuhException, match=f".* {expected_exc} .*"):
         Agent.unset_single_group_agent(agent_id, group_id, force=force)
 
+
+def test_check_uninstall_permission():
+    """Check that agent_check_uninstall_permission returns the expected msg"""
+
+    result = check_uninstall_permission()
+    expected = WazuhResult({'message': "User has permission to be uninstalled"})
```

Comment:
```suggestion
    expected = WazuhResult({'message': 'User has permission to uninstall agents'})
```

Reply:
Changed in b7dcc227e48732b77db2c92c1087e322a55d8ae9.